pkg/controller/job/job_controller_test.go

```diff
@@ -361,6 +361,19 @@ func TestControllerSyncJob(t *testing.T) {
 			expectedDeletions:       1,
 			expectedFailed:          1,
 		},
+		"WQ job: turn on PodReplacementPolicy but not set PodReplacementPolicy": {
+			parallelism:             1,
+			completions:             -1,
```

Comment:
This `completions=-1` looks suspicious, probably allowed because there is no validation in unit tests, but use realistic setup.

Reply:
We use completions -1 throughout this unit tests but I can use a different value.

Reply:
I see, please use a different value :)